docs/competition/volunteers/refreshments.md

```diff
@@ -0,0 +1,28 @@
+# Refreshments
+
+It's important that volunteers stay hydrated, which can be difficult for those in roles which don't allow them to move. (Don't give refreshments to competitors).
+
+*In SR2019, the refreshments are in the Volunteer Rest room, which is past the toilets and to the left.*
+
+It is the responsibility of the Refreshments role to offer and provide refreshments to all volunteers, especially those who can't leave their roles.
+
+To do this, you should grab a tray, fill it with a mix of drinks and food, and go to all volunteer positions, offering refreshments.
+Please also fill some cups with water from a nearby water fountain, and brew coffee/tea if available.
+
+Please don't forget about those at tech decks, those at reception/helpdesk, and those who are commentating/marshalling.
+
+## Running low on refreshments
+
+If we're running low on refreshments, it's your responsiblity to either go or find someone to go get more from the nearby shops.
```

Comment:
budget must be approved first

Reply:
Fixed in 49d401e

Reply:
Might a better approach be to pre-allocate a small amount of petty-cash for this, maybe even have that as cash stored somewhere to hand?

I agree we'd want to have someone check this, but I think we can mostly trust our volunteers and if we do run out we'll want to be able to replace it fairly promptly. If the treasurer is busy (quite likely given that we currently have a lot of role overlap) then getting explicit approval here could easily take a while.

Reply:
Yeah I'm happy to lay down £50 next to the refreshments and trust that people won't steal anything

Reply:
Fixed in eeb14dd1d7bdf94419f50f4ff571d01f970572cb and 3f0530c941d3a115a8c4c08c53a274b0533d24ae

Reply:
I would suggest we evaluate the refreshments levels at periodic points and decided to order more as we will have a good idea of how fast we are getting through them long before they run out.